components/Boards/PlayerVsPlayer.js

```diff
@@ -91,7 +90,16 @@ export default function RandomMoveEngine() {
       promotion: "q",
     });
     if (move === null) return false; // check illegal move
-    return true;
+
+    return true; // successful move
+  }
+
+  //set up payload on game over
+  if (game?.game_over()) {
+    console.log("Date: ", new Date().toLocaleString());
```

Comment:
Are these console logs for a future feature? I am not sure we want console logs in the `main` branch. Maybe you could display something on the page (even very basic) instead of console logging? i.e. a div that is conditionally rendered based on the value of `game?.game_over()`

Reply:
Ok, done.